internal/datastreams/pathway.go

```diff
@@ -26,7 +26,7 @@ func isWellFormedEdgeTag(t string) bool {
 	return false
 }
 
-func nodeHash(service, env string, edgeTags []string) uint64 {
+func nodeHash(service, env string, edgeTags []string, processTags []string) uint64 {
```

Comment:
```suggestion
func nodeHash(service, env string, edgeTags, processTags []string) uint64 {
```
For the sake of consistency